extensions/phylum.ts

```diff
@@ -74,11 +74,9 @@ async function requestHeaders(headersInit?: HeadersInit): Promise<Headers> {
   }
 
   // Set Content-Type header if it is missing.
-  if (!headers.has("Content-Type")) {
+  if (init.body && !headers.has("Content-Type")) {
```

Comment:
This has me wondering about the `{body: ""}` edge case... I have no idea whether that counts as a request with a body or not... But it's definitely not valid JSON, so I'm going to avoid that rabbit hole.

Reply:
`{body: ""}` won't get the header because `""` is not truthy. I think as far as HTTP requests are concerned, there isn't a difference between an empty body and no body. I considered checking if it was `null` or `undefined`, but the interface allows either and it's much easier to check if it's truthy than to check that it's not null and not undefined.

Reply:
> it's much easier to check if it's truthy than to check that it's not null and not undefined.

I wouldn't say "much easier". You can check for both null and undefined using `init.body == null` (thanks to loose equality)

But, as noted, `""` isn't valid JSON and may not even be a valid body. So setting `Content-Type: application/json` makes no sense for that request. The current code is correct.